### PR TITLE
Continue updating documentation

### PR DESCRIPTION
**Pull request by Soliloque**

### CODE REVIEW

File: manpages/fcpget.1 [hunk 1]
```diff
@@ -48,28 +63,21 @@ will be used unless overridden with '\-H'.
...
-.LP 
-.SH "AUTHORS"
-.LP
-The PyFCP applications and library were originally written
-by David McNab (aum) in May-June, 2006 and are maintained
-by the Freenet Project team.
+.SH "BUGS"
 .LP
-Contact the original author at: <davidmcnabnz@gmail.com>
+The support for asynchronous transfer is incomplete, making the
```
Comment (Thynix) [inline manpages/fcpget.1]: Oh! Is there a bug filed?

**Comment by Thynix** [comment]
Other than that this looks good.

File: manpages/fcpget.1 (hunk 1 above)
Comment (Soliloque) [inline manpages/fcpget.1]: No.

**Comment by Soliloque** [comment]
done!

**Comment by Thynix** [comment]
Thanks! I'm sorry for the delay - I was preparing for a flight to [CTS](https://openitp.org/festival/circumvention-tech-festival.html).

**Pull request merged**